releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -39,6 +39,12 @@ For more details on Telemetry, see [here](https://learn.microsoft.com/en-us/azur
 
 Added support for variant feature flags defined using [Microsoft Feature Management schema](https://github.com/microsoft/FeatureManagement/blob/c5fab16dbf1450dce0bbfe7c4207da735ff31916/Schema/FeatureManagement.v2.0.0.schema.json). Variants and telemetry can be declared using [Microsoft Feature Flag schema v2](https://github.com/microsoft/FeatureManagement/blob/c5fab16dbf1450dce0bbfe7c4207da735ff31916/Schema/FeatureFlag.v2.0.0.schema.json). Here is a [Sample](https://github.com/microsoft/FeatureManagement-Dotnet/blob/f47e188babea0a91488d2e6a0b2ab4c9405d0794/examples/VariantAndTelemetryDemo/appsettings.json#L12).
 
+#### Additional Changes
+
+* Optimized memory allocation for calls to `IsEnabledAsync` and `GetVariantAsync`.
+* Added support for .NET 9
+* Removed support for .NET 7
```

Comment:
This should be categorized under breaking change.

Reply:
And I don't think this belongs to the MS.FM library.

Reply:
For MS.FM- .net 7 was removed from tests. Meaning we are no longer ensuring there's not a .net 7 specific breaking change. 

For MS.FM.ASPNET- .net 7 was removed as a target from the actual library and from the tests. This means we no longer run tests against it and there's won't be a .net 7 specific build within the nuget package when installing MS.FM.ASPNET.  

There's no breaking change since the .net 6 build will continue to work in .net 7 environments.

Reply:
I'll add

* Removed support for .NET 7

In the ASP.NET library as well.

Reply:
This is releasing note of prod. Testing is irrelevant. Please remove.